python312Packages.txtai: 8.2.0 -> 8.5.0

GaetanLepage opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

Diff: https://github.com/neuml/txtai/compare/refs/tags/v8.2.0...refs/tags/v8.5.0
Changelog: https://github.com/neuml/txtai/releases/tag/v8.5.0

- **python312Packages.mcp: 1.6.0 -> 1.9.0**
- **python312Packages.fastapi-mcp: init at 0.3.4**
- **python312Packages.gliner: init at 0.2.20**
- **python312Packages.staticvectors: init at 0.2.0**
- **python312Packages.txtai: 8.2.0 -> 8.5.0**

cc @happysalada

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
python312Packages.txtai: 8.2.0 -> 8.5.0

Diff: https://github.com/neuml/txtai/compare/refs/tags/v8.2.0...refs/tags/v8.5.0

Changelog: https://github.com/neuml/txtai/releases/tag/v8.5.0

GaetanLepage committed:
python312Packages.fastapi-mcp: init at 0.3.4

GaetanLepage committed:
python312Packages.mcp: 1.6.0 -> 1.9.0

Diff: https://github.com/modelcontextprotocol/python-sdk/compare/refs/tags/v1.6.0...refs/tags/v1.9.0

Changelog: https://github.com/modelcontextprotocol/python-sdk/releases/tag/v1.9.0

GaetanLepage committed:
python312Packages.gliner: init at 0.2.20

GaetanLepage committed:
python312Packages.staticvectors: init at 0.2.0

josh commented:
Nice! @gaelj was trying to bump mcp in https://github.com/NixOS/nixpkgs/pull/409172 but we ran into some other dependency conflict.

GaetanLepage committed:
oterm: 0.11.1 -> 0.12.1

Diff: https://github.com/ggozad/oterm/compare/refs/tags/0.11.1...refs/tags/0.12.1

Changelog: https://github.com/ggozad/oterm/releases/tag/0.12.1

GaetanLepage committed:
python312Packages.fastmcp: init at 2.4.0

gaelj commented on pkgs/by-name/ot/oterm/package.nix:
```diff
@@ -9,36 +9,26 @@
-    "textual"
-    "typer"
-  ];
+  pythonRelaxDeps = true;
```
Thanks for updating this package as well. However I'd prefer to keep explicitly listing the relaxed deps. Thanks to this, my change to `mcp` broke the build of `oterm`, instead of building fine and releasing a broken version in terms of its mcp functionality. So we caught the problem early and I was able to request upstream adaptation of `oterm`.

GaetanLepage replied:
Ok fair enough!
